misc/applications/dispvm.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Name=QubesOS Open In DisposableVM
```

Comment:
Maybe call it "View in DisposableVM"? To be distinct from https://github.com/QubesOS/qubes-core-agent-linux/pull/310
And also please name the file more descriptively, maybe `qvm-view-in-dvm`?
But if `--view-only` is not necessary (IMO is), then that desktop file could be re-used.

Reply:
> But if `--view-only` is not necessary (IMO is), then that desktop file could be re-used.

This is actually not that obvious. I was thinking about the mail attachment case only, but with this being a generic feature, it does makes sense to preserve modifications made in the DispVM. And as for the attachment case, I believe Thunderbird will remove the temporarily saved file anyway (needs checking), so allowing modification shouldn't really matter.